pyaedt/hfss.py

```diff
@@ -2812,23 +2830,25 @@ def sar_setup(self, Tissue_object_List_ID=-1, TissueMass=1, MaterialDensity=1, v
 
         >>> oDesign.SARSetup
         """
-        self.odesign.SARSetup(TissueMass, MaterialDensity, Tissue_object_List_ID, voxel_size, Average_SAR_method)
+        self.odesign.SARSetup(tissue_mass, material_density, assignment, voxel_size, average_sar_method)
         self.logger.info("SAR Settings correctly applied.")
```

Comment:
```suggestion
        self.logger.info("SAR settings are correctly applied.")
```
Messages should be complete sentences, with verbs and concluding period.